CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# CHANGELOG
+
+Please enumerate all user-facing changes using format `<githib issue/pr number>: <short description>`, with changes ordered in reverse chronological order.
+
+## 1.0.0
+
+## 0.3.0
+
+#381: Bump maven-failsafe-plugin from 3.1.0 to 3.1.2
+#384: Bump guava from 32.0.0-jre to 32.0.1-jre
```

Comment:
Do we really want to include build/test scope dependency changes in the change log?

Reply:
I wondered about that too.  Where it gets tricky is that kroxylicious is publishing test artefacts .  I think we would want to include in those in the CHANGELOG.  I figured it was a greater mental load to have the changelog author filter correctly than it would be for an end-user skip by stuff that clearly wasn't pertinent.  Hence the default to all.

Reply:
Yeah fair. Maybe add something to clarify/remind about the test artefacts to the change log header/ pr template?

Reply:
Looks good, thanks @k-wall